lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1504,8 +1462,7 @@ def plot(self, xs, ys, *args, zdir='z', **kwargs):
 
     plot3D = plot
 
-    @_api.delete_parameter("3.4", "args", alternative="kwargs")
-    def plot_surface(self, X, Y, Z, *args, norm=None, vmin=None,
+    def plot_surface(self, X, Y, Z, norm=None, vmin=None,
```

Comment:
norm & co should stay keyword-only (they were already kwonly during the deprecation period)